app/start_handlers.py

```diff
@@ -666,10 +640,58 @@ async def get(self, request):
         }
 
 
-@start_routes.view('/start/timeout/')
-class UACTimeout(View):
-    @aiohttp_jinja2.template('timeout.html')
+@start_routes.view(r'/' + View.valid_display_regions + '/start/unlinked/address-has-been-linked/')
+class StartAddressHasBeenLinked(StartCommon):
+    @aiohttp_jinja2.template('start-unlinked-linked.html')
     async def get(self, request):
         self.setup_request(request)
-        self.log_entry(request, 'start/timeout')
-        return {}
+        await check_permission(request)
+        display_region = request.match_info['display_region']
+
+        if display_region == 'cy':
+            page_title = 'Your address has been linked to your code'
```

Comment:
if **page_title** is the same in both branches, why not just set it once above, or inline below ?

Reply:
If display_region is 'cy' the page_title should be in Welsh, however we haven't got Welsh text for lots of new content - adding the translations is a further piece of work (when all the English content is finalised). I put the if/else in everywhere as we will have translated titles eventually. Its why various content tests are the same in english and welsh as well

Reply:
ok fair enough